cilium/README.md

```diff
@@ -0,0 +1,65 @@
+# Agent Check: Cilium
+
+## Overview
+
+This check monitors [Cilium][1] through the Datadog Agent. The integration can either collect metrics from the `cilium-agent` or `cilium-operator`.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The Cilium check is included in the [Datadog Agent][2] package but requires additional setup steps to export Prometheus metrics.
+
+1. In order to collect `cilium-agent` metrics, the Prometheus metrics must be enabled by either:
+    * Deploy Cilium with the `global.prometheus.enabled=true` Helm value set.
+    * Add `--prometheus-serve-addr=:9090` to the `args` section of the Cilium DaemonSet config.
+        ```
+        [...]
+            spec:
+                containers:
+                - args:
+                    - --prometheus-serve-addr=:9090
+
+        ```
+2. In order to collect `cilium-operator` metrics, the Prometheus metrics must be enabled by adding `- --enable-metrics` to the args section of the Cilium DaemonSet config.
```

Comment:
just to clarify -- you can't enable this in Helm?

Reply:
Hi! Yes you can, I can also update with those instructions. The following were just the methods I used.